apps/server/src/modules/tldraw/metrics/metrics.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@nestjs/common';
+import { Gauge, register } from 'prom-client';
+
+@Injectable()
+export class MetricsService {
+	private numberOfUsersOnServerCounter: Gauge<string>;
+
+	private numberOfBoardsOnServerCounter: Gauge<string>;
+
+	constructor() {
+		this.numberOfUsersOnServerCounter = new Gauge({
+			name: 'tldraw_users',
```

Comment:
Please add the `sc_` prefix to the metric name - we add it for all the SchulCloud platform custom metrics.
```suggestion
			name: 'sc_tldraw_users',
```